src/fastcs/controller.py

```diff
@@ -29,6 +30,14 @@ def __init__(
 
         self._bind_attrs()
 
+    async def initialise(self) -> None:
```

Comment:
I don't like that we will always have to call super if this is overridden in a base class. It isn't really an override because it is doing a different thing - it is an implementation detail that should be hidden from the public API.

It would be nicer if we keep

```python
async def initialise(self):
    pass
```

to be overridden if needed, put this new logic in an `initialise_attributes` method and just have the backend call both

```python
controller.initialise()
controller.initialise_attributes()
``` 

What do you think?

Reply:
Yes I agree this would improve the public API.

Reply:
I've pushed the change as I think you would like it @GDYendell